src/app/countdown/countdown.module.ts

```diff
@@ -0,0 +1,24 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+
+import { CountdownDirective } from './directives/countdown.directive';
+import { TimeUntilPipe } from './pipes/time-until.pipe';
+import { SecondService } from './services/second.service';
+
+@NgModule({
+  imports: [
+    CommonModule
+  ],
+  declarations: [
+    TimeUntilPipe,
+    CountdownDirective,
+  ],
+  providers: [
+    SecondService
+  ],
+  exports: [
+    TimeUntilPipe,
+    CountdownDirective,
+  ]
+})
+export class CountdownModule { }
```

Comment:
One of my favorite novelties I added. This pattern lets us have live countdown timers in-app. I love it. To me it adds more life to the app when you can see a countdown. Also prevents someone from viewing an out-of-date expiration and thinking they have more time than they actually have.